Documentation/policy/language.rst

```diff
@@ -308,13 +308,9 @@ init
     security identity has not been resolved yet. This is typically only
     observed in non-Kubernetes environments. See section
     :ref:`endpoint_lifecycle` for details.
-health
-    The health entity represents the health endpoints, used to check cluster
-    connectivity health. Each node managed by Cilium hosts a health endpoint.
-    See `cluster_connectivity_health` for details on health checks.
```

Comment:
Spurious code changes are back :smile: 
It should be easier to remove them now that you have a single commit.